### PR TITLE
Enable llvm tests for symbolics

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**

### CODE REVIEW

**Comment by Thirumalai-Shaktivel** [comment]
Ready!

File: src/libasr/codegen/llvm_utils.cpp [hunk 1]
```diff
@@ -1267,6 +1267,7 @@ namespace LCompilers {
                 break;
             }
             case (ASR::ttypeType::CPtr) : {
+                a_kind = 8;
```
Comment (Thirumalai-Shaktivel) [inline src/libasr/codegen/llvm_utils.cpp:1270]: Previously, we used `a_kind = 4`, but now I set `8` to be default, this is required for Symbolic as we use `i64` pointer for `CPtr`.

Comment (certik) [reply src/libasr/codegen/llvm_utils.cpp:1270]: I thought CPtr doesn't have a kind?

Comment (Thirumalai-Shaktivel) [reply src/libasr/codegen/llvm_utils.cpp:1270]: How do we decide the malloc size for `list[CPtr]`?

Comment (anutosh491) [reply src/libasr/codegen/llvm_utils.cpp:1270]: I went through the changes and the changes do work in the correct direction. If we could know whether `a_kind` would be required or not for `list[CPtr]`, the PR can be finished accordingly .

Comment (anutosh491) [reply src/libasr/codegen/llvm_utils.cpp:1270]: cc @certik 
Could you clarify Thirumalai's comment here? I think this is close to completion once we have an idea for the same.

**Review by certik — APPROVED** [review]
I think this is good for now, it enables several tests and the changes are small and seem clean. I do not understand the design of a kind for `CPtr`, but that's a separate question. So I think we can merge this.

**Pull request merged**